packages/navigation/src/UIGridView/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { ListRenderItem, View, StyleSheet } from 'react-native';
+import { FlatList } from '..';
+import { UIConstant } from '../constants';
+import type { GridProps } from './types';
+
+export function UIGridView<T>({
+    data,
+    renderItem: renderItemProp,
+    containerStyle,
+    itemHeight,
+    testID,
+}: GridProps<T>) {
+    const [squareSize, setSquareSize] = React.useState({ width: 0, height: 0 });
+
+    const renderItem: ListRenderItem<T> = React.useCallback(
+        ({ item, index, separators }) => {
+            return (
+                <View style={[styles.item, squareSize]}>
+                    {renderItemProp?.({ item, index, separators })}
```

Comment:
I guess there is no need to make optional check since the prop is not optional

Reply:
It was optional